translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"immediately.\n"
+"\n"
+"\t\tNote that the delete command does NOT do resource version checks, so if "
+"someone\n"
+"\t\tsubmits an update to a resource right when you submit a delete, their "
+"update\n"
+"\t\twill be lost along with the rest of the resource."
+msgstr ""
+"\n"
+"\t\tDelete resources by filenames, stdin, resources and names, or by "
+"resources and label selector.\n"
+"\n"
+"\t\tJSON and YAML formats are accepted. Only one type of the arguments may "
+"be specified: filenames,\n"
+"\t\tresources and names, or resources and label selector.\n"
+"\n"
+"\t\tSome resources, such as pods, support graceful deletion. These resources "
+"define a default period\n"
+"\t\tbefore they are forcibly terminated (the grace period) but you may "
+"override that value with\n"
+"\t\tthe —grace-period flag, or pass —now to set a grace-period of 1. Because "
+"these resources often\n"
+"\t\trepresent entities in the cluster, deletion may not be acknowledged "
+"immediately. If the node\n"
+"\t\thosting a pod is down or cannot reach the API server, termination may "
+"take significantly longer\n"
+"\t\tthan the grace period. To force delete a resource,\tyou must pass a grace"
+"\tperiod of 0 and specify\n"
+"\t\tthe —force flag.\n"
+"\n"
+"\t\tIMPORTANT: Force deleting pods does not wait for confirmation that the "
+"pod’s processes have been\n"
+"\t\tterminated, which can leave those processes running until the node "
+"detects the deletion and\n"
+"\t\tcompletes graceful deletion. If your processes use shared storage or "
+"talk to a remote API and\n"
+"\t\tdepend on the name of the pod to identify themselves, force deleting "
+"those pods may result in\n"
+"\t\tmultiple processes running on different machines using the same "
+"identification which may lead\n"
+"\t\tto data corruption or inconsistency. Only force delete pods when you are "
+"sure the pod is\n"
+"\t\tterminated, or if your application can tolerate multiple copies of the "
+"same pod running at once.\n"
+"\t\tAlso, if you force delete pods the scheduler may place new pods on those "
+"nodes before the node\n"
+"\t\thas released those resources and causing those pods to be evicted "
+"immediately.\n"
+"\n"
+"\t\tNote that the delete command does NOT do resource version checks, so if "
+"someone\n"
+"\t\tsubmits an update to a resource right when you submit a delete, their "
+"update\n"
+"\t\twill be lost along with the rest of the resource."
+
+#: pkg/kubectl/cmd/stop.go:31
+msgid ""
+"\n"
+"\t\tDeprecated: Gracefully shut down a resource by name or filename.\n"
+"\n"
+"\t\tThe stop command is deprecated, all its functionalities are covered by "
+"delete command.\n"
+"\t\tSee 'kubectl delete --help' for more details.\n"
+"\n"
+"\t\tAttempts to shut down and delete a resource that supports graceful "
+"termination.\n"
+"\t\tIf the resource is scalable it will be scaled to 0 before deletion."
+msgstr ""
+"\n"
+"\t\tDeprecated: Gracefully shut down a resource by name or filename.\n"
```

Comment:
If this is deprecated maybe it doesn't exists anymore in the original file, so this can be removed.

Reply:
still in the file